autoload/ale/lsp.vim

```diff
@@ -260,8 +261,12 @@ function! s:UpdateCapabilities(conn, capabilities) abort
         let a:conn.capabilities.symbol_search = 1
     endif
 
-    if type(get(a:capabilities, 'workspaceSymbolProvider')) is v:t_dict
-        let a:conn.capabilities.symbol_search = 1
```

Comment:
Why are you deleting this? Is the `get(a:capabilities, 'workspaceSymbolProvider') is v:true` check above really enough?

Reply:
Didn't mean to delete this. Change reverted.